rules/windows/discovery_file_dir_discovery.toml

```
@@ -20,9 +27,16 @@ tags = ["Elastic", "Host", "Windows", "Threat Detection", "Discovery"]
 type = "eql"
 
 query = '''
-process where event.type in ("start", "process_started") and
-  (process.name : "cmd.exe" or process.pe.original_file_name == "Cmd.Exe") and
-  process.args : ("dir", "tree")
+sequence by process.entity_id with maxspan=1m
```

smart!

but these wont all have the same `process.entity_id`. maybe this?
```suggestion
sequence by agent.id, user.name with maxspan=1m
```